src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlObjectSerializerWriteContextComplex.cs

```diff
@@ -51,18 +51,31 @@ internal override bool WriteClrTypeInfo(XmlWriterDelegator xmlWriter, Type dataC
             return false;
         }
 
+#if NET_NATIVE
```

Comment:
My understanding of this change is that, regardless of the consolidation, we'd like to make this method internal in NetCore. Am I right? I just wondered why we need to have the method have different visibility in NetNative and NetCore while our goal is to consolidate them.